egs2/wsj_kinect/enh1/local/data.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Author: Atharva Anand Joshi (atharvaa@andrew.cmu.edu)
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+help_message=$(cat << EOF
+Usage: $0
```

Comment:
Could you add the documentation of the arguments below in this help message?

Reply:
Since the `min_or_max` and `sample_rate` are fixed for Kinect WSJ, data.sh does not have any arguments. Should the `help_message` part be removed?

Reply:
Added the documentation for the `parallel` and `use_dereverb` arguments.